Descent3/menu.cpp

```diff
@@ -1398,6 +1455,76 @@ int DisplayLevelWarpDlg(int max_level) {
   hwnd.Destroy();
   return chosen_level;
 }
+
+// DisplayLevelSelectDlg
```

Comment:
Unless the `DisplayLevelWarpDlg` function can be used elsewhere, it could be removed entirely since it is now unused. I haven't tested the cheat to warp while in a level though.